### PR TITLE
style DelegateCardList, minimum prettier tabWidth 2 config and settin…

### PR DESCRIPTION
**Pull request by ferrodri**
DelegateCardList styles migrated. I did also set a minimum prettier of 2 tabWidth and the .vscode/settings.json
screenshot

### CODE REVIEW

File: src/components/Delegates/DelegateCardList/DelegateCardList.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                  </VStack>
+                  <p className={styles.summary}>
+                    Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+                    Reiciendis ipsa dolor obcaecati, nisi beatae asperiores
+                    autem incidunt animi amet perspiciatis inventore enim fuga
+                    alias aut doloremque quibusdam exercitationem explicabo
+                    praesentium!
+                  </p>
+                  <div className={styles.flex_1} />
+                  <DelegateActions
```
Comment (kent) [inline src/components/Delegates/DelegateCardList/DelegateCardList.jsx:76]: Can you add in the Tiwtter and Discord links too, just stub them in for now. They come from something called the Delegate Statement. If you look at the code for `import { getStatment } from "@/app/api/statements/getStatements";` and `<DelegateStatement />` you can see where they are coming in from .

**Review by kent — CHANGES_REQUESTED** [review]
This is looking awesome Fernando! Great first day. Added in some comments.

File: src/components/Delegates/DelegateCardList/DelegateCardList.module.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        color: #66676b;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        line-clamp: 2;
+        -webkit-line-clamp: 2;
+        -webkit-box-orient: vertical;
+        font-size: $font-size-base;
+        line-height: $line-height-normal;
+      }
+      .flex_1 {
```
[the remark this answers is not in the document]
Comment (kent) [reply src/components/Delegates/DelegateCardList/DelegateCardList.module.scss:45]: Sorry to be clear I mean the empty `flex_1` div. Could use a Mixin too maybe?

[the remark this answers is not in the document]
Comment (ferrodri) [reply src/components/Delegates/DelegateCardList/DelegateCardList.module.scss:45]: Solved with tailwind

File: src/components/Delegates/DelegateCard/DelegateActions.jsx [hunk 3]
```diff
@@ -3,15 +3,26 @@ import { DelegateButton } from "./DelegateButton";
...
+  discord: "agora",
+  twitter: "agora",
+};
 export function DelegateActions({ className, address, votingPower }) {
   return (
     <HStack
       justifyContent="space-between"
       alignItems="stretch"
-      className={className}
+      className={className ? className + "justify-between" : "justify-between"}
```
Comment (ferrodri) [inline src/components/Delegates/DelegateCard/DelegateActions.jsx:15]: Hi Kent, justify-content: space-between was either not working on HStack so I had to add this line

File: src/components/Delegates/DelegateCardList/DelegateCardList.jsx (hunk 1 above)
Comment (ferrodri) [reply src/components/Delegates/DelegateCardList/DelegateCardList.jsx:76]: done!

**Review by kent — APPROVED** [review]

**Pull request merged**